tools/tune/tdfb/bf_design.m

```diff
@@ -20,11 +20,11 @@
 	error('The steer_az and steer_el lengths need to be equal.');
 end
 
-if length(find(bf.steer_az > 180)) || length(find(bf.steer_az < -180))
+if ~isempty(find(bf.steer_az > 180, 1)) || ~isempty(find(bf.steer_az < -180, 1))
 	error('The steer_az angles need to be -180 to +180 degrees');
 end
 
-if length(find(bf.steer_el > 90)) || length(find(bf.steer_el < -90))
+if ~isempty(find(bf.steer_el > 90, 1)) || ~isempty(find(bf.steer_el < -90, 1))
```

Comment:
What is the issue that ~isempty(find(),1) fixes? It's more complicated then length(find()). Also, the generic script fixes not related to design improvement should be in another commit.

Reply:
Maybe this would be nicest way to error with illegal angles `if max(bf.steer_az) > 180 || min(bf.steer_az) < -180`